File: src/main/java/com/petmate/server/dto/OrgMemberDto.java
package com.petmate.server.dto;

import lombok.Data;

@Data
public class OrgMemberDto {
    private Long id;
    private Long userId;
    private String userName;
    private String userEmail;
    private String userAvatarUrl;
    private String memberRole;
    private String status;
}

File: app/src/main/java/com/example/petmate/model/OrgMemberDto.kt
package com.example.petmate.model

data class OrgMemberDto(
    val id: Long? = null,
    val userId: Long? = null,
    val userName: String? = null,
    val userEmail: String? = null,
    val userAvatarUrl: String? = null,
    val memberRole: String? = null
)
